docs/RC/OnRCStatus/index.md

```diff
@@ -18,6 +18,8 @@ SDL must send OnRCStatus notification when:
     * A module is de-allocated from an application (the module is freed when application exits, disconnects, goes to HMI level NONE)
     * An application no longer has the right to control the module (for example, a policy update can revoke the application's access to a module)
 
+SDL must always include `moduleId` for each and every `ModuleData` in `allocatedModules` and `freeModules` in OnRCStatus notification.
```

Comment:
Maybe put this point inside a 
```markdown
!!! MUST   

!!!
``` 
block

Reply:
Thought the guidelines **!!! must !!!** sections describe **HMI** behaviour,  
I guess it might be misleading if I put this SDL note inside !!! must !!! section.

Reply:
You are correct. It would not apply for this case